test/integration/ctl/ctl_status_certificate_test.go

```diff
@@ -0,0 +1,187 @@
+	//kubeClient, _, cmCl, _ := framework.NewClients(t, config)
+	_, _, cmCl, _ := framework.NewClients(t, config)
+
+	const (
+		issuedAndUpToDateExpOutput = `Name: testcrt-1
+Namespace: testns-1
+Conditions:
+  Ready: True, Reason: , Message: Certificate is up to date and has not expired
+DNS Names:
+- www.example.com
+Issuer:
+  Name: letsencrypt-prod
+  Kind: ClusterIssuer
+Secret Name: example-tls
+Not After: 2020-09-16T09:26:18Z`
+
+		issuedAndDuringRenewalExpOutput = `Name: testcrt-2
+Namespace: testns-1
+Conditions:
+  Ready: True, Reason: , Message: Certificate is up to date and has not expired
+  Issuing: True, Reason: , Message: Issuance of a new Certificate is in Progress
+DNS Names:
+- www.example.com
+Issuer:
+  Name: letsencrypt-prod
+  Kind: ClusterIssuer
+Secret Name: example-tls
+Not After: 2020-09-16T09:26:18Z`
+	)
+
+	var (
+		crt1Name = "testcrt-1"
+		crt2Name = "testcrt-2"
+		ns1      = "testns-1"
+
+		readyAndUpToDateCond = cmapi.CertificateCondition{Type: cmapi.CertificateConditionReady,
+			Status: cmmeta.ConditionTrue, Message: "Certificate is up to date and has not expired"}
+		issuingCond = cmapi.CertificateCondition{Type: cmapi.CertificateConditionIssuing,
+			Status: cmmeta.ConditionTrue, Message: "Issuance of a new Certificate is in Progress"}
+	)
+
+	certIsValidTime, err := time.Parse(time.RFC3339, "2020-09-16T09:26:18Z")
+	if err != nil {
+		t.Fatal(err)
+	}
+	//certIsInvalidTime, err :=
+
+	crt1 := gen.Certificate(crt1Name,
+		gen.SetCertificateNamespace(ns1),
+		gen.SetCertificateDNSNames("www.example.com"),
+		gen.SetCertificateIssuer(cmmeta.ObjectReference{Name: "letsencrypt-prod", Kind: "ClusterIssuer"}),
+		gen.SetCertificateSecretName("example-tls"),
+	)
+
+	crt2 := gen.Certificate(crt2Name,
+		gen.SetCertificateNamespace(ns1),
+		gen.SetCertificateDNSNames("www.example.com"),
+		gen.SetCertificateIssuer(cmmeta.ObjectReference{Name: "letsencrypt-prod", Kind: "ClusterIssuer"}),
+		gen.SetCertificateSecretName("example-tls"),
+	)
+
+	tests := map[string]struct {
+		certificate       *cmapi.Certificate
+		certificateStatus *cmapi.CertificateStatus
+		inputArgs         []string
+		inputNamespace    string
+
+		expErr    bool
+		expOutput string
+	}{
+		"certificate issued and up-to-date": {
+			certificate: crt1,
+			certificateStatus: &cmapi.CertificateStatus{Conditions: []cmapi.CertificateCondition{readyAndUpToDateCond},
+				NotAfter: &metav1.Time{Time: certIsValidTime}},
+			inputArgs:      []string{crt1Name},
+			inputNamespace: ns1,
+			expErr:         false,
+			expOutput:      issuedAndUpToDateExpOutput,
+		},
+		"certificate issued and renewal in progress": {
+			certificate: crt2,
+			certificateStatus: &cmapi.CertificateStatus{Conditions: []cmapi.CertificateCondition{readyAndUpToDateCond, issuingCond},
+				NotAfter: &metav1.Time{Time: certIsValidTime}},
+			inputArgs:      []string{crt2Name},
+			inputNamespace: ns1,
+			expErr:         false,
+			expOutput:      issuedAndDuringRenewalExpOutput,
```

Comment:
Instead of declaring a var for `crt1/crt2` and `issueAndUpToDateExpOutput` and `issueAndDuringRenewalExpOutput`, can you just in-line them here? It'll make the tests *far* easier to read and understand, and save jumping back and forth.

Once we get up to say, 10 tests, this will be *extremely* difficult to accurately review 😬